fuse-products/src/main/java/software/tnb/product/integration/generator/IntegrationGenerator.java

```diff
@@ -158,7 +159,8 @@ public static void processCustomizers(AbstractIntegrationBuilder<?> integrationB
                 Container container = traits.getContainer() == null ? new Container() : traits.getContainer();
                 container.setServicePort((long) integrationBuilder.getPort());
                 traits.setContainer(container);
-            })
+            }),
+            new BytemanCustomizer()
```

Comment:
is the bytemancustomizer always added to every integration? is it safe?

Reply:
I'd say so, it does nothing when there are no rules: https://github.com/tnb-software/TNB/blob/f896885538206f033f6090540aca79ab8f03d473/fuse-products/src/main/java/software/tnb/product/customizer/byteman/BytemanCustomizer.java#L26-L28

Reply:
ohhh rules, great.